### PR TITLE
Forms in a detailed guide - content edits

### PR DESCRIPTION
**Pull request by JackHMRC**
Hi John, 

Can you look over this and see that it is published while I'm on leave please?

Cheers

Jack

### CODE REVIEW

File: src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk [hunk 1]
```diff
@@ -19,17 +19,22 @@ Use this pattern if users need to read detailed guidance before using the form,
...
+
+<p class="govuk-body">
+Types of form that you can use in a detailed guide include the following: 
 </p>
 
 <ul class="govuk-list govuk-list--bullet">
-<li>Interactive guidance forms — these are added as external links. The URL will begin with https://www.tax.service.gov.uk/guidance</li>
-<li>PDFs that can be completed on-screen (interactive) — these are added as file attachments and require Adobe to open and complete</li>
-<li>PDFs that cannot be completed on screen (not interactive) — these are added as file attachments and do not require Adobe to open</li>
+<li>Online services — These are built by service teams on the tax platform behind the Government Gateway, accessed with a green button with a URL, starting with https://www.taxservices.gov.uk/</li>
```
Comment (timsb) [inline src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk:34]: I've never seen `https://www.taxservices.gov.uk/` so I'm guessing it should actually be:
```suggestion
<li>Online services — These are built by service teams on the tax platform behind the Government Gateway, accessed with a green button with a URL, starting with https://www.tax.service.gov.uk/</li>
```

File: src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk [hunk 2]
```diff
@@ -40,6 +45,13 @@ Types of form that can be included in this format are:
 More than one type of form can be hosted on the same page. You can still only use the green button once on the page (check the <a href="/patterns-hmrc-govuk-team/green-button/" class="govuk-link">green button pattern</a> for more information). For example: <a href="https://www.gov.uk/guidance/register-as-a-bingo-promoter" class="govuk-link">Register as a bingo promoter</a>.
 </p>
 
+<h3 class="govuk-heading-m" id="Detailed examples">Detailed guidance examples</h3>
+<ul class="govuk-list govuk-list--bullet">
+  <li><a href="/guidance/register-as-a-bingo-promoter" class="govuk-link">Register as a bingo promoter</a> - an example of an online service with a green button and LC form to register by print and post</li>
```
Comment (timsb) [inline src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk:50]: `/guidance/register-as-a-bingo-promoter` which resolves to https://www.tax.service.gov.uk/guidance/register-as-a-bingo-promoter` doesn't exist guessing this should be `gov.uk`?
```suggestion
  <li><a href="https://www.gov.uk/guidance/register-as-a-bingo-promoter" class="govuk-link">Register as a bingo promoter</a> - an example of an online service with a green button and LC form to register by print and post</li>
```

File: src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk [hunk 3]
```diff
@@ -40,6 +45,13 @@ Types of form that can be included in this format are:
...
+  <li><a href="/guidance/claim-a-tax-refund-if-youve-stopped-work-and-flexibly-accessed-all-of-your-pension-p50z" class="govuk-link">Claim a tax refund if you've stopped work and flexibly accessed all of your pension (P50Z)</a> —  example of a service with a green button and both an interactive guidance (print and post) form and a PDF (print and post) form for digitally excluded users</li>
```
Comment (timsb) [inline src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk:51]: As previous comment. Guessing this should be:
```suggestion
  <li><a href="https://www.gov.uk/guidance/claim-a-tax-refund-if-youve-stopped-work-and-flexibly-accessed-all-of-your-pension-p50z" class="govuk-link">Claim a tax refund if you've stopped work and flexibly accessed all of your pension (P50Z)</a> —  example of a service with a green button and both an interactive guidance (print and post) form and a PDF (print and post) form for digitally excluded users</li>
```

File: src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk [hunk 4]
```diff
@@ -134,22 +146,67 @@ For any other formats, include the following text under the link to the form:
...
+<p class="govuk-body">
+  Some trials are taking place to look at user demand for existing PDFs. To support this a page owner may agree to move the different form choices (an existing PDF and online service) into an interactive guide away from detailed guide. 
+  </p>
+  
+  <p class="govuk-body">
+  Read ‘Instructions: forms in a detailed guide’ to find out when you can move form choices into and IG tool and how you must format content.
+  </p>
+  
+  <p class="govuk-body">
+  IG tools with form options must follow this pattern and other GOV.UK patterns for form choices — unless evidence of a strong user need demonstrates a reason not to.^
```
Comment (timsb) [inline src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk:191]: Guessing the caret (^) shouldn't be at the end of this sentence?
```suggestion
  IG tools with form options must follow this pattern and other GOV.UK patterns for form choices — unless evidence of a strong user need demonstrates a reason not to.
```

File: src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk [hunk 5]
```diff
@@ -134,22 +146,67 @@ For any other formats, include the following text under the link to the form:
...
+  </p>
+  
+  <p class="govuk-body">
+  The detailed guide must use a ‘single green button’ to send users into the IG tool. Do not use the Government Gateway user ID’ pattern. 
+  </p>
+  
+  <p class="govuk-body">
+  <h3 class="govuk-heading-s" id="formats">Example</h3> 
+  </p>
```
Comment (timsb) [inline src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk:200]: The `h3` shouldn't be wrapped in a paragraph.
```suggestion
  <h3 class="govuk-heading-s" id="formats">Example</h3> 
```

File: src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk [hunk 6]
```diff
@@ -222,11 +275,24 @@ Check with the subject matter expert if you need to include the following text:
...
+   </p>
+    
+    <ul class="govuk-list govuk-list--bullet">
+    <li>sit behind the Government Gateway (exceptions to this are rare) —  but they’re not the same as online services.</li>
+    <li>start with https://www.tax.service.gov.uk/submissions  —  not to the same as interactive guidance forms hosted in the '/guidance' directory of the tax service platform</li>
+    </ul>
+
+    <p class="govuk-body">
+      <h3 class="govuk-heading-s" id="formats">Example</h3> 
+      </p>
```
Comment (timsb) [inline src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk:291]: The `h3` shouldn't be wrapped in a paragraph.
```suggestion
      <h3 class="govuk-heading-s" id="formats">Example</h3> 
```

**Review by timsb — CHANGES_REQUESTED** [review]

**Review by Jon-Rowe-HMRC — APPROVED** [review]

**Review by Edem-Ag — APPROVED** [review]

**Review by timsb — APPROVED** [review]

**Pull request merged**